app/views/home/me.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="inset">
+ <h2>Welcome, <%= current_user.email_address %></h2>
+</div>
```

Comment:
Done.

I'd still like to keep the `/me` endpoint though, not least because I think it might be helpful longer term for UX to populate it with User details and link to from the header. No story for that yet, though.